lib/twilio/backoff.ts

```diff
@@ -9,11 +9,13 @@ class Backoff extends EventEmitter {
    * @param {object} options
    * @property {number} min - Initial timeout in milliseconds [100]
    * @property {number} max - Max timeout [10000]
-   * @property {boolean} jitter - Apply jitter [0]
+   * @property {number} jitter - Apply jitter [0]
    * @property {number} factor - Multiplication factor for Backoff operation [2]
    */
   constructor(options) {
     super();
+    this._min = options.min || 100;
+    this._isRetryAfterSet = false;
```

Comment:
Backoff module is an independent module that doesn't know anything about outside. We should not couple this module to any SDK logic so we can reuse it. Please add retryAfter logic in wstransport. Also, instead of boolean, add a private member in wstransport `_retryAfter`. If it's set, use it. If it's undefined, use the default backoff min.